import/xsl/dspace.xsl

```diff
@@ -19,7 +19,7 @@
                 <!-- ID -->
```

Comment:
Is there a reason for not including the collection handling here as is done in other files?

Reply:
Good catch! An oversight on my part. Without the collection handling, this XSLT created a separate <add> element for each record, so I think it looked correct to me when I tested (some of the other XSLTs would mash everything together into a single record). However, it was missing the top-level collection tag, so it was definitely not right without this logic.

Reply:
Ok, now it looks good to me!